src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_functionapp_commands.py

```diff
@@ -475,24 +474,26 @@ def test_functionapp_workloadprofiles(self, resource_group, storage_account):
             'wlp', 15
         )
 
-        self.cmd('containerapp env create --name {} --resource-group {} --location {} --enable-workload-profiles  --logs-destination none'.format(
+        self.cmd('containerapp env create --name {} --resource-group {} --location northeurope --enable-workload-profiles  --logs-destination none'.format(
             managed_environment_name,
             resource_group,
-            location,
         ))
-        
+
+        time.sleep(260)
```

Comment:
Why do we need such a long sleep time? In addition, do we only need to re-record these tests in live mode?
```suggestion
        if self.is_live:
              time.sleep(260)
```

Reply:
Sometimes it takes longer for the resources to be created so if we try to run commands with resources that have not yet finished getting created, we get errors. The Centauri workload profiles and dapr tests specially take a longer time to run that other tests.